examples/langchain-llama2-chatbot-sagemaker/README.md

```diff
@@ -0,0 +1,73 @@
+# Building a Llama2 Conversational Chatbot with AWS and LangChain
+
+In this guide, you'll discover how to create a simple conversational chatbot by integrating Pluto, AWS services, LangChain, and Llama2. We’ll walk you through the architecture components of our example application and how to deploy and operate it using Pluto.
+
+## Architecture Overview
+
+![Chatbot Architecture Diagram](../../assets/langchain-llama2-chatbot-sagemaker-arch.png)
+
+As illustrated above, our example application relies on the following AWS services and resources:
+
+- **Amazon SageMaker**: Deploying the Llama2 model
+  - Model, EndpointConfiguration, Endpoint, Role
+- **Amazon DynamoDB**: Persisting session messages
+  - Table
+- **AWS Lambda**: Executing backend business logic
+  - Function, Role, Policy
+- **Amazon API Gateway**: Providing an HTTP API endpoint for user access
+  - Api, Route, Integration, Deployment, Stage
+
+## Comparison
+
+The table below shows a comparison between Pluto and other tools regarding the amount and type of code:
```

Comment:
Shouldn't these be compared to Lepton and its competitors for application developers? After all, Pluto's positioning is not actually an IaC tool.